Tests/SubprocessTests/LinuxTests.swift

```diff
@@ -64,56 +63,31 @@ struct SubprocessLinuxTests {
     }
 
     @Test func testSuspendResumeProcess() async throws {
-        let result = try await Subprocess.run(
+        _ = try await Subprocess.run(
             // This will intentionally hang
             .path("/usr/bin/sleep"),
             arguments: ["infinity"],
-            output: .discarded,
             error: .discarded
-        ) { subprocess -> Error? in
-            do {
-                try await tryFinally {
-                    // First suspend the process
-                    try subprocess.send(signal: .suspend)
-                    try await waitForCondition(timeout: .seconds(30), comment: "Process did not transition from running to stopped state after $$") {
-                        let state = try subprocess.state()
-                        switch state {
-                        case .running:
-                            return false
-                        case .zombie:
-                            throw ProcessStateError(expectedState: .stopped, actualState: state)
-                        case .stopped, .sleeping, .uninterruptibleWait:
-                            return true
-                        }
-                    }
-                    // Now resume the process
-                    try subprocess.send(signal: .resume)
-                    try await waitForCondition(timeout: .seconds(30), comment: "Process did not transition from stopped to running state after $$") {
-                        let state = try subprocess.state()
-                        switch state {
-                        case .running, .sleeping, .uninterruptibleWait:
-                            return true
-                        case .zombie:
-                            throw ProcessStateError(expectedState: .running, actualState: state)
-                        case .stopped:
-                            return false
-                        }
-                    }
-                } finally: { error in
-                    // Now kill the process
-                    try subprocess.send(signal: error != nil ? .kill : .terminate)
+        ) { subprocess, standardOutput in
+            try await tryFinally {
+                // First suspend the process
+                try subprocess.send(signal: .suspend)
+                try await waitForCondition(timeout: .seconds(30)) {
+                    let state = try subprocess.state()
```

Comment:
Why change this? I found it necessary to check for the additional states in order to make the test more reliable.